js/views/userPageVw.js

```diff
@@ -303,10 +303,16 @@ UserPageVw = pageVw.extend({
             model.set('headerURL', self.options.userModel.get('serverUrl') + "get_image?hash=" + model.get('profile').header_hash + "&guid=" + self.pageID);
             model.set('avatarURL', self.options.userModel.get('serverUrl') + "get_image?hash=" + model.get('profile').avatar_hash + "&guid=" + self.pageID);
           }
+
           // Cache user avatar in localStorage
           var profile = model.toJSON().profile;
           window.localStorage.setItem("avatar_" + self.pageID, profile.avatar_hash);
 
+          if (response.profile.website) {
```

Comment:
1.) Why are there seemingly 3 different vars referring to the same thing? Can't we make this clearer via:

```
          if (profile.website) {
            self.short_website = self.trimUrl(profile.website);
          }
```

2.) Camel case seems to be our pretty standard naming convention, so can we update `short_website` to follow suit?

3.) What is the purpose of this short_website variable? All we end up doing with it is sending it to the template and it never gets used there.